mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/logging.kt

```diff
@@ -0,0 +1,65 @@
+package com.jakewharton.mosaic
+
+import com.github.ajalt.mordant.platform.MultiplatformSystem
+import kotlin.jvm.JvmField
+
+private const val MOSAIC_LOG_PATH = "MOSAIC_LOG_PATH"
+private const val MOSAIC_LOG_LEVEL = "MOSAIC_LOG_LEVEL"
+
+public enum class LogLevel(internal val envVarValue: String) {
+	VERBOSE(envVarValue = "verbose"),
+	DEBUG(envVarValue = "debug"),
+}
```

Comment:
I'm not sure about the logging levels and which ones are needed, especially considering the needs of the library and the needs of library users.